src/formatDate.js

```diff
@@ -48,9 +48,37 @@
  *
  * @returns {string}
  */
-
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
 
+  const calendar = {
+    formatYear(format) {
+      if (this.hasOwnProperty('YYYY') && format === 'YY') {
+        this['YY'] = this['YYYY'].slice(2, 4);
+        delete this['YYYY'];
+      } else if (this.hasOwnProperty('YY') && format === 'YYYY') {
+        this['YYYY'] = this['YY'] <= 20 ? '20' + this['YY'] : '19' + this['YY'];
```

Comment:
you can simplify it by using template strings literals:

![image](https://github.com/mate-academy/js_format_date_advanced/assets/30219274/150d8bfa-0fd3-4976-aee7-679b4beb6fed)